src/main/scala/tech/cryptonomic/conseil/tezos/TezosNodeOperator.scala

```diff
@@ -9,7 +9,7 @@ import tech.cryptonomic.conseil.util.CryptoUtil.KeyStore
 import tech.cryptonomic.conseil.util.JsonUtil.fromJson
 
 import scala.concurrent.{ExecutionContext, Future}
-import scala.util.{Failure, Try}
+import scala.util.Try
 
 /**
   * Operations run against Tezos nodes, mainly used for collecting chain data for later entry into a database.
```

Comment:
I guess `asyncGetAccountForBlock` is now a duplicate, since there's no sync version anymore

Reply:
Right, got the yellow bar in IntelliJ

Reply:
did you remove the duplicate?